Mage/Task/AbstractTask.php

```diff
@@ -202,7 +202,8 @@ protected function getParameters()
         $localCommand = 'ssh ' . $this->getConfig()->getHostIdentityFileOption() . $needs_tty . ' -p ' . $this->getConfig()->getHostPort() . ' '
             . '-q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no '
             . $this->getConfig()->getConnectTimeoutOption()
-            . $this->getConfig()->deployment('user') . '@' . $this->getConfig()->getHostName();
+            . ( $this->getConfig()->deployment('user') != '' ? $this->getConfig()->deployment('user') . '@' : '' )
```

Comment:
Ok, I agree ;) Personally, I like to keep consistency of writing parentheses even if they are purely for grouping purposes.